packages/sei-cosmwasm/README.md

```diff
@@ -57,6 +57,37 @@ Currently, Sei Bindings support query and message support for the sei custom mod
             - Change the Admin of the Denom. Only the current admin can change the admin.
         - SetMetadata
             - Set the denom metadata of a factory denom. Only the current admin can set metadata.
+- EVM
+    - Query
+        - StaticCall
+            - TODO
```

Comment:
not sure what this does

Reply:
this is basically the "generic query" endpoint (i.e. query any endpoint on any EVM contract)